app/src/main/java/cm/aptoide/pt/home/BottomNavigationActivity.java

```diff
@@ -60,4 +65,13 @@
       }
     }
   }
+
+  private Event getStoreEvent() {
+    Event event = new Event();
+    event.setAction("https://ws75.aptoide.com/api/7/getStoreWidgets/store_id=15/context=stores");
```

Comment:
I am not sure if we should keep this this way. It seems really ugly to hardcode the event needed to launch the fragment, but I do not have an alternative aswell, maybe we should keep the changes for the future when we turn this fragment to full mvp.